docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -53,6 +53,35 @@ updates.
 
 ## Enroll instructions
 
+<Details
+scope={["cloud"]}
+scopeOnly
+opened
+title="Finding Agents">
+
+You can find agents that need to be enrolled using `tctl inventory ls` like so:
```

Comment:
```suggestion
opened
title="Finding Agents in Teleport Enterprise Cloud">

If you have a Teleport Enterprise Cloud account, you can find agents that need to be enrolled using `tctl inventory ls` like so:
```

Otherwise, the vast majority of users will miss these instructions. The default scope is `oss`, and users tend not to switch scopes.